### PR TITLE
Several improvements (from Asana list)

### PR DESCRIPTION
**Pull request by pferreir**
* Author sorting
* Markdown in comments
* Fixed judgment comment placeholder
* Added CfA announcement on conference home page

### CODE REVIEW

File: indico/modules/events/abstracts/models/comments.py [hunk 1]
```diff
@@ -60,7 +67,7 @@ class AbstractComment(db.Model):
         index=True,
         nullable=False
     )
-    text = db.Column(
+    _text = db.Column(
```
Comment (ThiefMaster) [inline indico/modules/events/abstracts/models/comments.py]: `'text'` as the first argument so it doesn't expect the column to be named `_text`

File: indico/modules/events/abstracts/models/reviews.py [hunk 2]
```diff
@@ -77,7 +84,7 @@ class AbstractReview(db.Model):
         UTCDateTime,
         nullable=True
     )
-    comment = db.Column(
+    _comment = db.Column(
```
Comment (ThiefMaster) [inline indico/modules/events/abstracts/models/reviews.py]: `'comment'` for the column name

File: indico/modules/events/abstracts/__init__.py [hunk 3]
```diff
@@ -141,3 +142,10 @@ def _reviewing_area_visible(conf):
     yield MenuEntryData(title=_("Reviewing Area"), name='user_tracks', endpoint='abstracts.display_reviewable_tracks',
                         position=0, parent='call_for_abstracts',
                         visible=_reviewing_area_visible)
+
+
+@template_hook('conference-home-info')
+def _inject_regform_announcement(event, **kwargs):
```
Comment (ThiefMaster) [inline indico/modules/events/abstracts/__init__.py]: regform? :monkey:

File: indico/modules/events/abstracts/__init__.py [hunk 4]
```diff
@@ -141,3 +142,10 @@ def _reviewing_area_visible(conf):
     yield MenuEntryData(title=_("Reviewing Area"), name='user_tracks', endpoint='abstracts.display_reviewable_tracks',
                         position=0, parent='call_for_abstracts',
                         visible=_reviewing_area_visible)
+
+
+@template_hook('conference-home-info')
+def _inject_cfa_announcement(event, **kwargs):
+    event = event.as_event
+    if event.cfa.is_open or (session.user and event.cfa.can_submit_abstracts(session.user)):
```
Comment (ThiefMaster) [inline indico/modules/events/abstracts/__init__.py]: check whether the feature is enabled first. `can_submit_abstracts` always triggers a DB query to get the list of authorized users if the cfa is not open

File: indico/web/templates/forms/_person_link_widget_base.html [hunk 5]
```diff
@@ -15,12 +15,13 @@
 {% endmacro %}
 
 {% block html %}
-    {% set custom_display_order = field._value()|selectattr('displayOrder')|list|length > 0 %}
+    {% set list_in_custom_order = (field._value()|selectattr('displayOrder')|list|length > 0) %}
```
Comment (OmeGak) [inline indico/web/templates/forms/_person_link_widget_base.html]: Out of curiosity. Why is `(...)` needed here?

Comment (ThiefMaster) [inline indico/web/templates/forms/_person_link_widget_base.html]: I don't think it is. Probably just to make it prettier (I tend to do the same when having kwargs with operators)

Comment (pferreir) [inline indico/web/templates/forms/_person_link_widget_base.html]: It's not needed. Probably left over from a previous version where I was joining that clause with something else.

**Pull request closed without merging**